experimental/rocm/tracing.c

```diff
@@ -37,8 +50,24 @@ struct iree_hal_rocm_tracing_context_t {
 
   // Event pool reused to capture tracing timestamps.
   hipEvent_t event_pool[IREE_HAL_ROCM_TRACING_DEFAULT_QUERY_CAPACITY];
+  
+  SmuDumpInitFunc f_smuDumpInit;
```

Comment:
put these on their own struct so they are logically grouped - you can then drop the f prefix (as this file will never include anything) - `struct { SmuDumpInitFunc smuDumpInit; ... } smutrace;`